src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -7,9 +7,10 @@ unreleased
 * `vmss get-instance-view`: Fix issue where extra, erroneous information was displayed when using `--instance-id *`
 * `vmss create`: Added support for `--lb-sku`
 * `vm/vmss create`: remove human names from the admin name blacklist
-* `vm/vmss create`: fix issue where the command would throw an error if unable to extract plan information from an image. 
+* `vm/vmss create`: fix issue where the command would throw an error if unable to extract plan information from an image.
 * `vmss create`: fix a crash when create a scaleset with an internal LB
 * `vm availability-set create`: Fix issue where --no-wait argument did not work.
+* Fix documentation with --ids 
```

Comment:
History entries and not desired/required for help text changes.

Reply:
I added it to follow required Pull request guidelines 
`The PR has modified HISTORY.rst with an appropriate description of the change`

should I removed it

Reply:
Yes, please remove it. I've created a PR to update our PR template :)